agent/src/IngestDetailsRepository.php

```diff
@@ -74,7 +74,7 @@ private function refresh(): PromiseInterface
         $start = microtime(true);
         $duration = null;
 
-        return $this->browser->post('/api/agent-auth', headers: [], body: '')
+        return $this->browser->post('/api/agent-auth', headers: [], body: '{}')
```

Comment:
I've set the browser's `content-type` to JSON. Figure we should send a valid, even if empty, payload.

Reply:
Dig it. Updated.